webknossos/examples/download_image_data.py

```diff
@@ -15,7 +15,7 @@ def main() -> None:
 
     # Download the datset
     ds = wk.Dataset.download(
-        dataset_name=name,
+        dataset_name_or_url=name,
```

Comment:
Do we consider a change of a parameter name to be a breaking change?

Reply:
Good point, yes :+1: I'll add this to the changelog.